administrators.rst

```diff
@@ -516,6 +517,27 @@ Fields:
   2.0 system.
 * **Region**: the region in Swift. This field is optional.
 
+.. _amazon-s3:
+
+S3 (Amazon)
+^^^^^^^^^^^
+
+Amazon S3 is available as an access protocol as of Storage Service 0.12. At this
+time, locations within S3 have only been tested for AIP Storage.
+
+Fields:
+
+* **Size**: the maximum size allowed for this space. Set to 0 or leave blank
+  for unlimited. This field is optional.
+* **Path**: the absolute path to the Space on the local filesystem.
+* **Staging path**: the absolute path to a staging area. Must be UNIX filesystem
+  compatible and preferably will be located on the same filesystem as the path.
+* **S3 Endpoint URL**: the URL of the S3 endpoint, e.g.
+  ``https://s3.amazonaws.com``.
+* **Access Key ID to authenticate**: the public key generated by S3.
+* **Secret Access Key to authenticate with**: the secret key generated by S3.
+* **Region**: the region that the S3 instance uses, e.g. ``us-east-2``
```

Comment:
The period is missing at the end of the sentence.

Reply:
:sob: